tests/handlers/test_oidc.py

```diff
@@ -591,6 +598,63 @@ def test_exchange_code(self):
             yield defer.ensureDeferred(self.handler._exchange_code(code))
         self.assertEqual(exc.exception.error, "some_error")
 
+    @defer.inlineCallbacks
+    def test_extra_attributes(self):
+        """
+        Login while using a mapping provider that implements get_extra_attributes.
+        """
+        # We cannot just override the configuration since the mapping provider
+        # gets stored on the handler.
```

Comment:
I'm not really following this, tbh.

Reply:
My point was that I could not get `override_config` to work. I think this is because the mapping provider gets stored on the handler during instantiation. Although if a new homeserver is being built, a new handler should...I think.

Reply:
I was able to fix this, it was a bug in how the initial OIDC config was generated for tests. Thanks for calling it out!